Kai.BaseModule/KaiSDK.cs

```diff
@@ -561,5 +575,24 @@ private static void DecodeConnectedKais(JObject input)
 				};
 			}
 		}
+
+		private static void DecodeKaiConnected(JObject input)
+		{
+			var kaiID = input[Constants.KaiId].ToObject<int>();
+			var hand = input[Constants.Hand]?.ToObject<string>();
+			var defaultKai = input[Constants.DefaultKai]?.ToObject<bool>();
+			
+			if (!Enum.TryParse(hand, true, out Hand handEnum))
+				handEnum = Hand.Left;
+			
+			connectedKais[kaiID] = new Kai
+			{
+				KaiID = kaiID,
+				Hand = handEnum
+			};
+
+			if (defaultKai == true)
+				DefaultKai = connectedKais[kaiID];
```

Comment:
This line would delete the `Default` reference which will end up losing all the events that the user has subscribed to. The `Default` object has to remain constant and cannot change.

Reply:
Ok, so just set Default.KaiId and Default.Hand

Reply:
Yes, exactly